### ISSUE TITLE
Further parsing is needed for power functions

### ISSUE THREAD

**Issue by chadcwilliams** [AutoResearch/equation-tree#2]
The power functions are all treated the same; however, it would be better to distinguish them as squared, cubed, square root, etc. Otherwise, the equation discovery approaches will treat the power as a coefficient and this would not accurately reflect the prior.

**Comment by younesStrittmatter** [comment]
sqrt is already included as function. We could add squared and cubed to the functions, but we would need to parse this as a special case when reading sympy functions.

**Comment by chadcwilliams** [comment]
Yeah. The fix would be to check the child of the power node for the exponent. And I agree we should only to this up to cubed. So, we have sqrt, squared, cubed, power in general.
